service-api/module/Application/src/Model/Service/ProcessingStatus/Service.php

```diff
@@ -178,10 +185,37 @@ private function handleResponse(ResponseInterface $result)
         if (isset($responseBody['rejectedDate'])){
```

Comment:
Make a private method to reduce duplication of code.

Reply:
On looking at this again, there's not a good way I can think of (short of passing both arrays into a function) to only set a key on $return if it is also set in $responseBody, so I'm going to leave this as-is.